ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -84,3 +104,35 @@ export const getSummariesTable = (summaries: UnitSummary[]) => {
   }
   return groupedUnits
 }
+
+export const unitSummariesTable2 = (summaries: UnitsSummary[]) => {
+  const unitSummaries = summaries.map((unitSummary) => {
+    return getSummaryRow(
+      unitSummary.minimumIncomeMin,
+      unitSummary.minimumIncomeMax,
+      unitSummary.monthlyRentMin,
+      unitSummary.monthlyRentMax,
+      unitSummary.monthlyRentAsPercentOfIncome || "",
+      unitSummary.monthlyRentAsPercentOfIncome || "",
+      unitSummary.totalAvailable,
+      unitSummary.totalCount,
+      unitSummary.unitType.name
+    )
+  })
+
+  return unitSummaries
+}
+
+export const getSummariesTable2 = (summaries: UnitsSummary[]) => {
```

Comment:
I want to learn more about overloading React function components (so that we can have the same name for this function and for `getSummariesTable`). In the meantime, though, can we give these two functions names that differentiate them more? (maybe something like `getSummariesTableFromUnitsSummaries` and `getSummariesTableFromUnitSummaries`, though admittedly those two are _very_ similar and that's also confusing...)

Reply:
There's probably a way better way to do this, but one possibility is:
- Define `unitsSummariesTable` like `unitsSummariesTable = (summaries: UnitSummary[] | UnitsSummary[])`
- Delete `unitsSummariesTable2`
- Make the call to `getSummaryRow` with the correct args based on the type provided.
- Delete `getSummariesTable2`

WDYT?

Reply:
Gave it a try here.  Getting lots of warnings about types but it seems to be working??
![image](https://user-images.githubusercontent.com/82653098/131395304-88a03709-0886-4f3a-be6c-5af7aab31633.png)

Reply:
So after debugging together, we believe there isn't a way to let the functions know which type of data they have (UnitSummary or UnitsSummary).  As a result I've pulled the two functions out again, and given them more appropriate names.